Mark automatically redirected windows damaged when their contents change.
Otherwise compPaintWindowToParent() skips copying their backing pixmaps
to the parent window.

--- a/Xext/composite/compalloc.c
+++ b/Xext/composite/compalloc.c
@@ -89,7 +89,7 @@ compReportDamage(DamagePtr pDamage, RegionPtr pRegion, void *closure)
         QueueWorkProc(compScreenUpdate, serverClient, pScreen);
         cs->pendingScreenUpdate = true;
     }
-    cw->damaged = false;
+    cw->damaged = true;

     compMarkAncestors(pWin);
 }
